### PR TITLE
Limit PR must be reviewed check to pytorch org

### PR DESCRIPTION
**Pull request by malfet**
And add `ciflow/trunk` on merges only to `pytorch/pytorch`.

Tweak tests quite a lot to distinguish between the two scenarios.

### CODE REVIEW

File: torchci/lib/bot/pytorchBotHandler.ts [hunk 1]
```diff
@@ -226,22 +227,25 @@ The explanation needs to be clear on why this is needed. Here are some good exam
...
-    let rejection_reason = null;
 
-    if (forceRequested) {
-      rejection_reason = await this.reasonToRejectForceRequest(forceMessage);
-    } else {
-      // Ensure the PR has been signed off on
-      let approval_status = await this.getApprovalStatus()
-      if (approval_status !== PR_APPROVED) {
-        rejection_reason = "This PR needs to be approved by an authorized maintainer before merge."
+    if (this.owner == "python") {
```
Comment (clee2000) [inline torchci/lib/bot/pytorchBotHandler.ts:231]: typo?

Comment (clee2000) [inline torchci/lib/bot/pytorchBotHandler.ts:231]: typo?

Comment (huydhn) [inline torchci/lib/bot/pytorchBotHandler.ts:231]: I think you mean `pytorch` here, lol

**Review by clee2000 — APPROVED** [review]

File: torchci/lib/bot/pytorchBotHandler.ts (hunk 1 above)
Comment (malfet) [reply torchci/lib/bot/pytorchBotHandler.ts:231]: Thank you very much for spotting the typo. Yes, only pytorch, so I really hope it's going to be caught by testing...

**Review by huydhn — APPROVED** [review]
A small typo and some tests need to be updated, otherwise LGTM

**Pull request merged**